packages/core/test/plugin-progress-bar.config.mock.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * This config file is here to demonstrate the progress bar for plugins
+ *
+ * Usage:
+ * npx ./dist/packages/cli collect --config=./packages/core/test/plugin-progress-bar.config.mock.ts
+ */
+
+const outputDir = 'tmp';
+const pluginSlug = (id: string): string => 'async-plugin-' + id;
+const auditSlug = (pId: string, aId: string): string =>
+  pluginSlug(pId) + '-a' + aId;
+const pluginTitle = (end: string): string => 'Async Plugin ' + end;
+const auditTitle = (end: string): string => 'Async Audit ' + end;
+const asyncPlugin = (pId: string, duration = 1000) => {
+  const aId = '1';
+  const outputFile = `${outputDir}/${pluginSlug(pId)}-output.json`;
+  return {
+    slug: pluginSlug(pId),
+    title: pluginTitle(pId),
+    icon: 'javascript',
+    audits: [{ slug: auditSlug(pId, aId), title: auditTitle(aId) }],
+    runner: {
+      command: 'node',
+      args: [
+        '-e',
+        `setTimeout(() => require('fs').writeFileSync('${outputFile}', '${JSON.stringify(
```

Comment:
As discussed on Slack, this is likely to fail on Windows shells, so we should test it works on Windows first before merging this (hopefully will be part of CI checks soon once #166 is merged).

The simplest workaround I can think of is to invoke the runner as a separate Node script instead of inlining the code (sidesteps all the complexity of how different shells handle quotes and split commands).

Reply:
done